.github/workflows/sync-query-zone.yml

```
@@ -0,0 +1,45 @@
+name: sync query zone
```

```suggestion
name: Sync query zone
```
Just a little nit-pick.